utils/portworx_configure_max_storage_node_per_zone.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+# Check if portworx helm charts are installed and the chart status
+# helm history portworx (Find the status etc)
+
+# if installed, then configure the max storage node per zone
+
+# Wait for the pods to be restarted
+function version { printf "$@" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }'; }
+
+NAMESPACE=$3
+PX_CLUSTER_NAME=$2
+NODE_COUNT=$1
+SLEEP_TIME=30
+
+DIVIDER="\n*************************************************************************\n"
+HEADER="$DIVIDER*\t\tConfigure Requested to Portworx Enterprise ${IMAGE_VERSION}\t\t*$DIVIDER"
+
+DESIRED=0
+READY=0
+JSON=0
+
+# Install helm3
+# Check the Helm Chart Summary
+printf "[INFO] Kube Config Path: $CONFIGPATH"
+export KUBECONFIG=$CONFIGPATH
+kubectl config current-context
+
+CMD="helm"
+VERSION=$($CMD version | grep v3)
+if [ "$VERSION" == "" ]; then
+    printf "[WARN] Helm v3 is not installed, migrating to v3.3.0..."
+    mkdir /tmp/helm3
+    wget https://get.helm.sh/helm-v3.3.0-linux-amd64.tar.gz -O /tmp/helm3/helm-v3.3.0-linux-amd64.tar.gz
+    tar -xzf /tmp/helm3/helm-v3.3.0-linux-amd64.tar.gz -C /tmp/helm3/
+    CMD="/tmp/helm3/linux-amd64/helm"
+    $CMD version
+fi
+
+# Get the portworx chart namespace
+CHART_NAMESPACE=$(helm ls -A | grep portworx | awk '{print $2}')
+# Get the Helm status
+if ! JSON=$(helm history portworx -n ${CHART_NAMESPACE} -o json | jq '. | last'); then
+    printf "[ERROR] Helm couldn't find Portworx Installation, will not proceed with the upgrade!! Please install portworx and then try to upgrade.\n"
```

Comment:
Change above ERROR msg.
```suggestion
    printf "[ERROR] Helm couldn't find Portworx Installation, will not proceed with configuring max storage nodes per zone!!\n"
```